app/src/main/java/com/example/dogbook/main/PostReactions.java

```diff
@@ -0,0 +1,87 @@
+package com.example.dogbook.main;
```

Comment:
nit: might be good for this to live under `data` or common/data but otherwise seems good!